kuma/static/styles/minimalist/components/_auth-providers.scss

```diff
@@ -6,6 +6,15 @@
         margin: 60px auto;
     }
 
+    &:focus {
+        outline-style: none;
+    }
+
+    &:focus-visible {
+        outline: 5px auto #e59700; // linux and windows
+        outline-color: -webkit-focus-ring-color; // everyone else
+    }
```

Comment:
Thanks so much for doing this @mindy. In this instance, perhaps it is better to just remove that outlined style on the modal entirely. For one thing, the browser support for `:focus-visible` is not great yet https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#Browser_compatibility, although generally I would be in favor of adding it anyhow.

Perhaps it is not needed on the modal itself as much seeing that the interaction flow suggested that, that is where the focus is moved. The other thing is `stylelint` does not know about the pseudo class(yet) and so linting fails(https://travis-ci.org/mdn/kuma/jobs/640659152#L293). We can get around that by disabling `stylelint` for this block, but perhaps all of that is simply not needed.

We do have proper focus states on the buttons as well as the close icon when they receive focus, and once the modal is open, tab order is trapped inside the modal so, we should be good.

What are your thoughts?

Reply:
I think that's a good call, especially since, as you mentioned, we have proper focus states on the elements within the modal. 

I upgraded stylelint to the latest version (13.0.0) so we should be okay (at least running it locally) if we do want to add the focus-visible class in the future!